submissions/sqrt-minus-one/main.cpp

```diff
@@ -13,6 +14,13 @@
 #include <thread>
 #include <x86intrin.h>
 
+#define TEST_WITH_GPERF 0
+
+#if TEST_WITH_GPERF
+#include "gperf_cities.h"
```

Comment:
Is this part of your submission ?

Reply:
No, that was part of local test I did to compare gperf performance to my custom hashing function, I decided to continue with my custom hashing so that code does nothing now.

Reply:
I added the files that were generated by gperf tool to git, though they are not compiled into the translation unit unless the TEST_WITH_GPERF is set to 1.

Reply:
I am not sure if your file will still run with the runner ? would the headers be loader ?

Reply:
No, the `#include` part under the `#if TEST_WITH_GPERF` will be skipped when compiling the file since `TEST_WITH_GPERF` evaluates to 0 so it shouldn't be loaded.